frontend/src/components/EnquiryForm.vue

```diff
@@ -75,11 +47,14 @@ export default {
       .then((res) => {
         switch (res.status) {
           case 202:
-            this.messageSent = true;
+            this.body = "";
+            this.email = "";
+            this.name = "";
```

Comment:
This doesn't clear the validation, so you're left with error messages when the form is submitted. Use the `reset()` method instead, as this will clear value and validation. Check this out: https://stackoverflow.com/questions/51059402/reset-vuetify-form-validation

Reply:
Thanks! for catching that out